### PR TITLE
[ET-1840] Attendees Page: Restyle Ticket Overview

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket

[ET-1840] 

### 🗒️ Description

We are re-styling the attendee page. This PR is for the `Ticket Overview` column.

### 🎥 Artifacts 

image

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1840]: https://theeventscalendar.atlassian.net/browse/ET-1840?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/template-tags/tickets.php [hunk 1]
```diff
@@ -493,7 +493,7 @@ function tribe_events_has_tickets_on_sale( $event_id ) {
 	 *
 	 * @return string
 	 */
-	function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $ticket ) {
+	function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $ticket, string $sold_label_override = '' ) {
```
Comment (redscar) [inline src/template-tags/tickets.php:496]: `@since` should be updated

**Review by redscar — COMMENTED** [review]
I think a snapshot test would be great for the new overview.

File: src/template-tags/tickets.php (hunk 1 above)
Comment (codingmusician) [reply src/template-tags/tickets.php:497]: Done.

Comment (codingmusician) [reply src/template-tags/tickets.php:497]: Also created a snapshot test that checks the entire summary on the attendees page.

**Review by redscar — APPROVED** [review]

**Review by redscar — DISMISSED** [review]

**Review by juanfra — COMMENTED** [review]
Looking good! I left some comments.

I'm wondering if we should avoid having the `-left` part on the icon file names.

Probably having `icon-series-pass`, `icon-rsvp`, and `icon-single-ticket` would make them more reusable and generic.

File: src/admin-views/attendees/attendees-event/overview.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+<div class="welcome-panel-column welcome-panel-middle">
+	<h3 class="tec-tickets__admin-attendees-overview-title">
+		<?php echo esc_html_x( 'Ticket Overview', 'attendee screen summary', 'event-tickets' ); ?>
+	</h3>
+	<?php do_action( 'tribe_events_tickets_attendees_ticket_sales_top', $event_id ); ?>
+	<?php foreach ( $tickets_by_types as $type_name => $type_tickets ) : ?>
+		<div class="tec-tickets__admin-attendees-overview-ticket-type">
+			<div class="tec-tickets__admin-attendees-overview-ticket-type-icon <?php echo esc_attr( $type_icon_class[ $type_name ] ) ?>"></div>
+			<div class="tec-tickets__admin-attendees-overview-ticket-type-label">
+				<?php esc_html_e( $type_name ) ?>
```
[the remark this answers is not in the document]
Comment (juanfra) [reply src/admin-views/attendees/attendees-event/overview.php:63]: I believe we should address this.

File: src/admin-views/attendees/attendees-event/overview.php [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+			foreach ( $type_tickets as $ticket ) {
+				$ticket_name = sprintf( '%s [#%d]', $ticket->name, $ticket->ID );
+				?>
+				<li class="tec-tickets__admin-attendees-overview-ticket-type-list-item">
+					<div>
+						<span class="tec-tickets__admin-attendees-overview-ticket-type-list-item-ticket-name">
+							<?php esc_html_e( $ticket->name ); ?>
+						</span>
+						<span class="tec-tickets__admin-attendees-overview-ticket-type-list-item-ticket-id">
+							<?php esc_html_e( sprintf( '#%d', $ticket->ID ) ); ?>
```
[the remark this answers is not in the document]
Comment (juanfra) [reply src/admin-views/attendees/attendees-event/overview.php:79]: I believe we should address this.

File: src/admin-views/attendees/attendees-event/summary.php [hunk 4]
```diff
@@ -76,33 +77,9 @@
...
-				<ul>
-					<?php
-					/** @var Tribe__Tickets__Ticket_Object $ticket */
-					foreach ( $tickets as $ticket ) {
-						$ticket_name = sprintf( '%s [#%d]', $ticket->name, $ticket->ID );
-						?>
-						<li>
-							<strong><?php echo esc_html( $ticket_name ) ?>:&nbsp;</strong><?php
-							echo esc_html( tribe_tickets_get_ticket_stock_message( $ticket ) );
+			<?php $this->template( 'attendees/attendees-event/overview', [ 'tickets' => $tickets ] ); ?>
```
[the remark this answers is not in the document]
Comment (juanfra) [reply src/admin-views/attendees/attendees-event/summary.php:81]: I believe we should address this.

File: src/resources/postcss/tickets-admin/attendees/_table.pcss [hunk 5]
```diff
@@ -29,4 +29,94 @@
...
+
+	.tec-tickets__admin-attendees-overview-ticket-type-list-item {
+		display: flex;
+		justify-content: space-between;
+		margin-right: 0;
+	}
+
+	.tec-tickets__admin-attendees-overview-ticket-type-list-item-ticket-name,
+	.tec-tickets__admin-attendees-overview-ticket-type-list-item-ticket-id {
+		font-size: 13px;;
```
[the remark this answers is not in the document]
Comment (juanfra) [reply src/resources/postcss/tickets-admin/attendees/_table.pcss:89]: I believe we should address this.

**Review by juanfra — COMMENTED** [review]
Can we please open a new PR to address the changes from the review?

**Pull request merged**